pyjsonviewer/pyjsonviewer.py

```diff
@@ -164,6 +164,7 @@ def find(self, search_text):
         self.collapse_all(None)
         for item_id in self.get_all_children(self.tree):
             item_text = self.tree.item(item_id, 'text')
+            item_text = str(item_text)
```

Comment:
why this change is needed?

Reply:
Thanks for prompt review.

I had some numeric values in my data. Search was breaking while applying `.lower()` on numeric data.
So I am converting all fields into `str`.
I am adding test cases for both the changes.